RecoLocalCalo/HGCalRecProducers/python/hgcalLayerClusters_cff.py

```diff
@@ -17,3 +17,17 @@
 hgcalLayerClusters.plugin.fcPerEle = cms.double(fC_per_ele)
 hgcalLayerClusters.plugin.noises = cms.PSet(refToPSet_ = cms.string('HGCAL_noises'))
 hgcalLayerClusters.plugin.noiseMip = hgchebackDigitizer.digiCfg.noise
+
+
+hgcalLayerClustersHFNose = hgcalLayerClusters_.clone(
+    detector = cms.string('HFNose'),
+    timeOffset = hfnoseDigitizer.tofDelay,
+    plugin = cms.PSet(
+        dEdXweights = cms.vdouble(dEdX.weightsNose),
+        fcPerMip = cms.vdouble(HGCalUncalibRecHit.HGCHFNoseConfig.fCPerMIP),
```

Comment:
type specs for all existing parameters better be dropped for automated syntax checking.

```python
detector = 'HFNose',
plugin = dict(
  dEdXweights = dEdX.weightsNose,
  ...
```